### PR TITLE
Disable Select Boundary If Below Min Zoom + Lower Boundary Min Zoom

### PR DESCRIPTION
**Pull request by arottersman**
## Overview

Before you were able to use the boundary selection draw tools even if the boundary layer you were selecting couldn't be accessed at the current zoom level. This PR fixes the disabling of the selection tools on zoom, closes out active tools if a zoom event makes them inaccessible, and decreases the min zoom for the boundary layers so they can be accessed at higher altitudes.

Connects #2217 

### Demo

ch94ywf8xn

## Testing Instructions

 * Pull this branch, and `bundle`
* Zoom around the map with select boundary tool open, and confirm layers are disabled when you zoom out passed their min zoom level
* Select a boundary tool and zoom way out — it should deselect when you zoom past its min zoom

### CODE REVIEW

File: src/mmw/js/src/draw/templates/drawTool.html [hunk 1]
```diff
@@ -13,18 +13,24 @@
         <ul class="dropdown-menu draw-pane-list">
             {% for item in toolData.items %}
                 <li class="draw-pane-list-item" role="presentation">
+                    <button class="draw-pane-list-item-button" role="button" tabindex="-1"
```
Comment (arottersman) [inline src/mmw/js/src/draw/templates/drawTool.html]: Except for the disable styling the button should look identical as to when it was a `<span>`

**Comment by kellyi** [comment]
Taking a look at this!

**Comment by kellyi** [comment]
Mentioned this but it looks like the button disappears on zooming past the valid level, but when you zoom back in the previously selected grid reappears although the button doesn't:

grid-boundaries

**Comment by kellyi** [comment]
Aside from the quirk noted above this is working well! I tested it out with congressional districts in Chrome, FF, Safari, and IE11.

**Comment by arottersman** [comment]
Thanks for catching that, @kellyi! I've pushed a fixup to correct.

**Comment by arottersman** [comment]
Thanks. Retargeted and squashed. Going to merge when the build completes.

**Comment by rajadain** [comment]
Hmm this seems to have merged a couple of extra things by accident (since it wasn't rebased on top of `release/1.20.0`)

**Pull request merged**